src/optimalcontrol/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(idyntree_optimalcontrol)
+
+set(libraryname idyntree-optimalcontrol)
+
+
+idyntree_enable_cxx11()
+
+option(BUILD_SHARED_LIBS "Build libraries as shared" TRUE)
+
+# Dependencies
+# IPOPT
+# SUNDIALS (?)
```

Comment:
I guess in the end you went for implementing your own integrator, we can remove this comments.